### ISSUE TITLE
SegmentCleanup.cleanup raised an exception on latest 1.8.0b5

### ISSUE THREAD

**Issue by JustMe0815** [roflcoopter/viseron#144]
This is what i am gettings sometimes and everything stops working.

```
[2021-04-07 10:05:00] [lib.recorder.frontdoor  ] [DEBUG   ] - Start event: 1617786278, segment: 20210407100438.mp4
[2021-04-07 10:05:00] [lib.recorder.frontdoor  ] [DEBUG   ] - End event: 1617786298, segment: 20210407100457.mp4
[2021-04-07 10:05:00] [lib.recorder.frontdoor  ] [DEBUG   ] - Concatenation command: ['ffmpeg', '-hide_banner', '-loglevel', 'error', '-y', '-protocol_whitelist', 'file,pipe', '-f', 'concat', '-safe', '0', '-i', '-', '-c:v', 'copy', '-movflags', '+faststart', '/recordings/2021-04-07/FrontDoor/10:04:48.mp4']
[2021-04-07 10:05:00] [lib.recorder.frontdoor  ] [DEBUG   ] - Segment script:
file '/recordings/segments/FrontDoor/20210407100438.mp4'
inpoint 0
file '/recordings/segments/FrontDoor/20210407100444.mp4'
file '/recordings/segments/FrontDoor/20210407100447.mp4'
file '/recordings/segments/FrontDoor/20210407100449.mp4'
file '/recordings/segments/FrontDoor/20210407100451.mp4'
file '/recordings/segments/FrontDoor/20210407100453.mp4'
file '/recordings/segments/FrontDoor/20210407100455.mp4'
file '/recordings/segments/FrontDoor/20210407100457.mp4'
outpoint 1
[2021-04-07 10:05:01] [lib.recorder.frontdoor  ] [DEBUG   ] - Segments concatenated
[2021-04-07 10:05:01] [lib.cleanup             ] [DEBUG   ] - Resuming segment cleanup
[2021-04-07 10:06:50] [apscheduler.executors.default] [ERROR   ] - Job "SegmentCleanup.cleanup (trigger: interval[0:00:05], next run at: 2021-04-07 09:06:55 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/src/viseron/lib/cleanup.py", line 85, in cleanup
    segment.split(".")[0], "%Y%m%d%H%M%S"
  File "/usr/lib/python3.7/_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib/python3.7/_strptime.py", line 359, in _strptime
    (data_string, format))
[2021-04-07 10:06:55] [apscheduler.executors.default] [ERROR   ] - Job "SegmentCleanup.cleanup (trigger: interval[0:00:05], next run at: 2021-04-07 09:07:00 UTC)" raised an exception, message repeated 2 times
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/src/viseron/lib/cleanup.py", line 85, in cleanup
    segment.split(".")[0], "%Y%m%d%H%M%S"
  File "/usr/lib/python3.7/_strptime.py", line 577, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/lib/python3.7/_strptime.py", line 359, in _strptime
    (data_string, format))
ValueError: time data '' does not match format '%Y%m%d%H%M%S'
```

**Comment by roflcoopter** [comment]
Thanks for the report! Will take a look

**Comment by roflcoopter** [comment]
This has been resolved in 1.8.0
